semantic/flatbuffers_test.go

```diff
@@ -292,178 +404,142 @@ func getFnExprFlatBuffer() (*semantic.Package, []byte) {
 	fbsemantic.FunctionExpressionAddLoc(b, exprLoc)
 	fbsemantic.FunctionExpressionAddTyp(b, funTy)
 	fbsemantic.FunctionExpressionAddTypType(b, fbsemantic.MonoTypeFun)
+	if funcExpOffset != 0 {
+		fbsemantic.FunctionExpressionAddVectorized(b, funcExpOffset)
+	}
 	fe := fbsemantic.FunctionExpressionEnd(b)
 
-	str := b.CreateString("f")
-	idLoc = getFBLoc(b, "1:1", "1:2", src)
-	fbsemantic.IdentifierStart(b)
-	fbsemantic.IdentifierAddLoc(b, idLoc)
-	fbsemantic.IdentifierAddName(b, str)
-	id := fbsemantic.IdentifierEnd(b)
+	return fe
+}
 
-	pt := getFnPolyType(b)
-	asnLoc := getFBLoc(b, "1:1", "1:36", src)
-	fbsemantic.NativeVariableAssignmentStart(b)
-	fbsemantic.NativeVariableAssignmentAddLoc(b, asnLoc)
-	fbsemantic.NativeVariableAssignmentAddTyp(b, pt)
-	fbsemantic.NativeVariableAssignmentAddIdentifier(b, id)
-	fbsemantic.NativeVariableAssignmentAddInit_(b, fe)
-	fbsemantic.NativeVariableAssignmentAddInit_type(b, fbsemantic.ExpressionFunctionExpression)
-	nva := fbsemantic.NativeVariableAssignmentEnd(b)
+func funcExpression(fe *semantic.FunctionExpression) *semantic.FunctionExpression {
```

Comment:
This function could do with a comment that describes what the argument is for, as it won't be obvious to someone reading the code.

Reply:
comments added